tests/ui/rust-2024/safe-outside-extern.rs

```diff
@@ -0,0 +1,20 @@
+safe fn foo() {}
```

Comment:
You should use revisions instead of duplicating this test, I think

```suggestion
//@ revisions: gated ungated
#![cfg_attr(gated, feature(unsafe_extern_blocks))]

safe fn foo() {}
```